src/config/media/pipewire.md

```diff
@@ -26,6 +23,66 @@ directory:
 # ln -s /usr/share/applications/pipewire.desktop /etc/xdg/autostart/pipewire.desktop
 ```
 
+## Session Management
+
+PipeWire requires a session manager to function. The reference
+[`pipewire-media-session`](https://gitlab.freedesktop.org/pipewire/media-session)
+package was originally provided and, in Void, configured to run by default to
+satisfy this requirement. However, `pipewire-media-session` is deprecated and
+the authors recommend using
+[WirePlumber](https://pipewire.pages.freedesktop.org/wireplumber/) in its place.
+Install the `wireplumber` package to use this session manager with PipeWire.
+
+The standard Void configuration, which causes `pipewire` to launch
+`pipewire-media-session` automatically, must be overridden to use `wireplumber`.
+The only change necessary is to comment out the `pipewire-media-session`
+invocation in the `context.exec` section, which can be done with a single `sed`
+substitution. To make this configuration change visible to all users, place the
+new configuration file in `/etc/pipewire`:
+
+```
+# mkdir -p /etc/pipewire
+# sed '/path.*=.*pipewire-media-session/s/{/#{/' \
+    /usr/share/pipewire/pipewire.conf > /etc/pipewire/pipewire.conf
+```
+
+Alternatively, place the new configuration file in the expected location for a
+single user:
+
+```
+$ : "${XDG_CONFIG_HOME:=${HOME}/.config}"
+$ mkdir -p "${XDG_CONFIG_HOME}/pipewire"
+$ sed '/path.*=.*pipewire-media-session/s/{/#{/' \
+    /usr/share/pipewire/pipewire.conf > "${XDG_CONFIG_HOME}/pipewire/pipewire.conf"
+
+```
+
+Now, configure `wireplumber` to start in the same fashion as the `pipewire`
+executable. The `wireplumber` package includes a Desktop Entry file that can be
```

Comment:
It is the opposite. wireplumber must be launched after pipewire otherwise wireplumber will exit.